packages/devtools_app/lib/src/main.dart

```diff
@@ -87,7 +87,17 @@ class HtmlPerfToolFramework extends HtmlFramework {
       queryId('try-flutter-web-devtools')
         ..hidden = false
         ..onClick.listen((_) {
-          // TODO(kenz): launch flutter web app.
+          String href = '/flutter.html#/';
+          // Preserve query parameters when opening the Flutter demo so the
+          // user does not need to go through the connect dialog again.
+          final flutterQueryParams =
+              Uri.tryParse(html.window.location.href).queryParameters ?? {};
+          if (flutterQueryParams.isNotEmpty) {
+            href += Uri(
+              queryParameters: flutterQueryParams,
```

Comment:
nit: I don't think the trailing comma adds much value here. Perhaps remove it since there is only one parameter and this isn't flutter build method code

Reply:
Done.